ui.apps/src/main/content/jcr_root/apps/aem-dictionary-translator/components/dictionary/dictionary.html

```diff
@@ -1,11 +1,13 @@
 <tr data-sly-use.model="be.orbinson.aem.dictionarytranslator.models.Dictionary"
     is="coral-table-row"
     data-foundation-collection-item-id="${resource.path}"
-    data-dictionary-item-editable="${model.editable ? 'true' : 'false'}"
     data-foundation-collection-navigator-href="/tools/translation/dictionaries/message-entries.html${resource.path}"
+    data-dictionary-publish-paths="${resource.path}"
```

Comment:
paths, but it's only path?

Reply:
consistency to cover in the action, same js is used to do replication